platforms/core-runtime/build-state/build.gradle.kts

```diff
@@ -18,24 +18,17 @@ plugins {
     id("gradlebuild.distribution.implementation-java")
 }
 
-description = "Types for build state management"
+description = "Types for build process and session state management"
 
 dependencies {
     api(project(":core"))
     api(project(":base-services"))
     api(project(":java-language-extensions"))
-    api(project(":core-api"))
```

Comment:
Moved some service registration back to core, where the services are implemented.